source/src_pdiag/pdiag_double.cpp

```diff
@@ -95,6 +95,8 @@ inline int set_elpahandle(elpa_t &handle, int *desc, int local_nrows, int local_
   elpa_set_integer(handle, "process_col", mypcol, &error);
 
   elpa_set_integer(handle, "blacs_context", desc[1], &error);
+
+  elpa_set_integer(handle, "cannon_for_generalized", 0, &error);
```

Comment:
This option seems to assign a method for elpa solver. Will this change affect performance?